portfolio/src/main/webapp/load-index.js

```diff
@@ -1,6 +1,18 @@
 // Set the nametag's HTML to be that fetched from the server
 async function getNametagText() {
   const response = await fetch('/nametag-text');
-  const greeting = await response.text();
-  document.getElementById('nametag').innerHTML = greeting;
+  const greetings = await response.json();
+
+  // Properly format and place the fetched greetings within the nametag
+  const nametag = document.getElementById('nametag'); 
+  nametag.innerHTML = formatComments(greetings);
+}
+
+// Format a list of comments such that their is a newline between each
+function formatComments (comments) {
+  master = '';
+  comments.forEach((comment) => {
+    master += comment + '</br>'
```

Comment:
It's preferable to wrap this text in a block element, such as `div`, `p` to force the newline, as opposed to `<br>`. You get better semantic meaning out of the HTML. Forcing whitespace and styling via HTML is not ideal, and you should try to push style handling into CSS.

You have much more control over whitespace with CSS box model, with control over margin, padding, and borders between paragraphs. https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model

Reply:
fixed! I also adjusted the margins of my nametag paragraphs to better fit the container